README.md

```diff
@@ -61,6 +61,24 @@ var env = multiplex.engines.nunjucks;
 env.addGlobal('key', 'value');
 ```
 
+## Cross-engine Embedding
+
+### Nunjucks
+
+To embed a template, use an `embed` filter in the parent template. Remember to pass it through `|safe` so it's not escaped.
```

Comment:
By default,  this will instantiate an environment with `autoescape: true`. It will be able to use custom environments in #5 

I added language to that effect in the readme.